### ISSUE TITLE
Make fundraising for this project.

### ISSUE THREAD

**Issue by zerocool1992** [REGoth-project/REGoth#356]
The Gothic game is very popular in the world. And your project very important to Gothic 1 and 2 games.
but not many people know about this project.
I think we must tell about this project on Gothic forums.
Also information how people donate to support project  can be added to readme.md

**Comment by mdrost** [comment]
I suggest bountysource https://www.bountysource.com/teams/regoth/issues

**Comment by Marek33** [comment]
Interesting site, but I would suggest Patreon, since its way more popular, and probably more people would be willing to donate, since a lot of people knew that site. I and probably a lot of other people have never seen that site, which you have mentioned.

**Comment by Jurigag** [comment]
I would suggest opencollective more cuz this is mostly used for programming projects.

@Marek33 bountysource is actually very common in programming world

**Comment by Marek33** [comment]
@Jurigag Maybe in the programming world, but not much outside, where most of the original Gothic fans are, who mostly have no idea, that there is such a huge project in the making. I'm saying this, since I see it all the time on the YouTube.

@zerocool1992 That would be probably the best thing, and also having multiple options: Patreon, Bountysource and PayPal.

**Comment by frabert** [comment]
Disclaimer: no the author, I do not speak on behalf of any other person other than me.

With that being said, I see some problems with this kind of support: what is needed in the development of this project is not money, IMO, but time, and that cannot be bought. The only way someone could afford to spend more time on this project is if it were to abandon his/her dayjob/studies and go fulltime on this, which would need a very substancial Patreon/whatever backing, which I do not think is going to be reached.

Also, who should receive the money, and how should it be spent? There are not many things money could be spent on right now, apart from maybe VR gear that was mentioned in #355, but I don't see it as a top priority

**Comment by Gryxx** [comment]
@frabert Keep in mind that you can hire someone to work on particular feature.

**Comment by mdrost** [comment]
Bounties for specific issue encourages new developers to look at the project.

**Comment by Gryxx** [comment]
@frabert Good point. But still, I think that this money could be well used (maybe not ASAP). I understand that it is not your decision to make, I just want to share some ideas (like that gret one from @mdrost ).

**Comment by mdrost** [comment]
I would like to share with you how well is other open source reimplementation project doing on bountysource https://www.bountysource.com/teams/openra/issues

**Comment by Marek33** [comment]
@frabert I fully agree with you on the time since that's the most important thing. However, I think that people in the community can wait for a mod/project to be completely finished since there are multiple mods, which are over 8 years in development.

Everyone who contributed to the Project in medium way at least, but especially the Andre, who is one of the main developers.

@Jurigag  You are right, but what if someone huge like Gopher (I doubt it since he doesn't really know Gothic unfortunately), would make a video about the REGoth and DirectX 11 mod for Gothic 1 & 2. Let's say, it would somehow get over one million of views and then it would be great to have a Patreon. I know that's very unlikely, but not impossible.

**Comment by Jurigag** [comment]
@Marek33 as i wrote, patreon is bad because it's not transparent. People can use opencollective as well, doesn't see any problem with it and it's good site too.

**Comment by Marek33** [comment]
@Jurigag On the Patreon, you can clearly see, on which the money will be used (if it's written there). However, I'm not sure when I could find it on that site. I think it would be best to have both options, since some people prefer Patreon and some Opencollective or Bountysource.

**Comment by Jurigag** [comment]
It's not written, for what exactly what amount of money goes. Only what people will write what on what they will spend, but they spend it in reality on anything else. On opencollective which each spend you need to type for what it's going so it's much more transparent.

**Comment by Marek33** [comment]
@Jurigag You are right about that, but I think that Andre knows what he is doing.

Indeed, its very interesting and the Patreon should have something like this too.

Yes, it is, but I dont think, its as popular as the Patreon, so thats why i suggested it.

I know, since ReGoth is opensourse, of the zEngine.

**Comment by zerocool1992** [comment]
The project must give a very simple donation  way to fans. Because if the donation will require eg. user registrations or some in same kind then many people won't donate.

**Comment by Marek33** [comment]
@zerocool1992 That's why I'm suggesting Patreon, since a lot of people in the YouTube are using it. So, there is a high chance that at least some ReGoth fans, have it. However, I don't know any site, where can I give money without registering. Maybe PayPal?

**Comment by zerocool1992** [comment]
Even simple payment requisites for anonymous donations in Readme.md can be useful for project.
Also it is impotent to add bitcoin, paypal, and similar payment addresses.
also there a lot of gothic fans in Russia, Germany, Poland etc. and local national online payment systems also need to be added like webmoney, yandexmoney, qiwi etc. to make donations very simple to fans.
eg. a can give to this project for example 20$ and i will not do this  with Patreon or similar platform. because i do not have an account. and it is difficult for me to create account.

**Comment by Jurigag** [comment]
But this is not youtube project. It's mainly if not only programming project and that's why we should use opencollective because it's more transparent. But whatever, i won't just support it if it will be on patreon because i won't know for what my money goes.

**Comment by frabert** [comment]
Discussion is appreciated, but it should be kept calm and non-personal please :) No need to get all fussed up

**Comment by Marek33** [comment]
@frabert  Don't worry, I have no intention to make any problems, but since there are multiple options, it would be better, if the people could choose, instead of having one or two options only.

**Comment by Jurigag** [comment]
I think it's better to have only one option because we will know how much exactly team have money and what they are doing with it, when splitting it we don't know and there is no transparency.
